src/pages/billing/models/AzureManagedAppCoordinates.ts

```diff
@@ -0,0 +1,7 @@
+export interface AzureManagedAppCoordinates {
+  tenantId: string // UUID as string
+  subscriptionId: string // UUID as string
+  managedResourceGroupId: string
+  region?: string
+  applicationDeploymentName?: string
```

Comment:
Do we ever have an applicationDeploymentName?

Reply:
This is in `src/pages/billing/CreateAzureBillingProjectModal.js`:
```
h(Select, {
            id,
            isMulti: false,
            placeholder: 'Select a managed application',
            isDisabled: errorFetchingManagedApps !== false || !!subscriptionIdError,
            value: chosenManagedApp,
            onChange: ({ value }) => setChosenManagedApp(value),
            options: _.map(application => {
              return {
                value: application,
                label: !!application.region ?
                  `${application.applicationDeploymentName} (${application.region})` :
                  application.applicationDeploymentName
              }
            }, managedApps)
          })
```